v2/pkg/protocols/common/expressions/expressions.go

```diff
@@ -53,3 +52,21 @@ func evaluate(data string, base map[string]interface{}) (string, error) {
 	// Replacer dynamic values if any in raw request and parse  it
 	return replacer.Replace(data, dynamicValues), nil
 }
+
+func findMatches(data string) []string {
+	var matches []string
+	tokens := strings.Split(data, "{{")
+	for _, token := range tokens {
```

Comment:
I think the logic in the `for` can be simplified:

```go
for _, token := range tokens {
  closingToken := strings.LastIndex(token, "}}")
  if closingToken > 0 {
    matches = append(matches, token[:closingToken])
  }
}
```

Also the `{{` and `}}` delimiters should be extracted to a global variable to be reused across the project (`replacer.go`, `expressions.go`, `maps.go`, `page_actions.go`).

Reply:
done